src/services/api/webforms/ABP2C/v202401.ts

```diff
@@ -304,6 +303,9 @@ export const v202401: FormSchema = {
               label:
                 "Other information about enrollment assurances for mandatory participants (optional)",
               labelClassName: "font-bold text-black",
+              props: {
+                className: "min-h-[114px]",
```

Comment:
Will this be the new default min height for text-area fields? Should we change that within the Slot as a pre-passed in style, or is this just for this form? I know you had talked with HCD about it.

Reply:
Great question, I think we need to talk to HCD more. We went through a lot of the webform HCD requested changes and they seem a bit random in heights and widths.

Reply:
If they can help us understand the standard they're using for field heights (assuming there is one), I agree it would be great if we could pull these styles out of the JSON and into the Slot